packages/css/src/unordered-list/unordered-list.scss

```
@@ -0,0 +1,31 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-unordered-list {
+  color: var(--amsterdam-unordered-list-color);
+  font-family: var(--amsterdam-unordered-list-font-family);
+  font-size: var(--amsterdam-unordered-list-font-size-narrow);
+  font-weight: var(--amsterdam-unordered-list-font-weight);
+  line-height: var(--amsterdam-unordered-list-line-height);
+  list-style: var(--amsterdam-unordered-list-list-style);
+  margin-block: var(--amsterdam-unordered-list-margin-block);
```

This should be 0, right? Components shouldn't set their own margins

---

The design shows an 8px margin top and bottom, shall I change it to this?

```suggestion
  padding-block: var(--amsterdam-unordered-list-padding-block);
```

---

No, it should be zero, components shouldn't set their own margins